src/__tests__/custom-document/__fixtures__/custom-document-with-gip/pages/_app.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export default function CustomApp({ Component, pageProps }) {
+  return (
+    <>
+      _app
+      <Component {...pageProps} />;
+    </>
+  );
+}
+
+CustomApp.getInitialProps = async (appContext) => {
+  return {
+    pageProps: {
+      ctx: appContext,
+      fromCustomApp: true,
+      propNameCollision: 'from-app',
+    },
```

Comment:
Probably we don't want to test that custom App is called with expected props here. Do you think we can safely remove these props?